bitbots_vision/cfg/Vision.cfg

```diff
@@ -26,45 +26,44 @@ obstacle_detector_enum = gen.enum([ gen.const("convex",      str_t, "convex", "f
 
-fcnn_paths = os.listdir(os.path.join(package_path, "models"))
-fcnn_paths_enum = gen.enum([gen.const(path, str_t, path, "fcnn %s" % path) for path in fcnn_paths], "An enum to select a fcnn model")
+dummy_list = ["dummy"]
+
+fcnn_paths_enum = gen.enum([ gen.const("dummy",      str_t, "dummy", "dummy decription")], "Dummy")
+yolo_paths_enum = gen.enum([ gen.const("dummy",      str_t, "dummy", "dummy decription")], "Dummy")
 
 group_vision = gen.add_group("vision", type="tab")
 group_ROS = gen.add_group("ROS", type="tab")
-group_ball_fcnn = gen.add_group("ball_fcnn", type="tab")
-group_ball_finder = gen.add_group("ball_finder", type="tab")
-group_field_boundary_finder = gen.add_group("field_boundary_finder", type="tab")
+group_neural_networks = gen.add_group("Neural Networks", type="tab")
 group_detector = gen.add_group("detector", type="tab")
 group_dynamic_color_space = gen.add_group("DynamicColorSpace", type="tab")
 
 group_color_detector = group_detector.add_group("color_detector", type="tab")
 group_line_detector = group_detector.add_group("line_detector", type="tab")
+group_field_boundary_detector = group_detector.add_group("field_boundary_detector", type="tab")
 group_obstacle_detector = group_detector.add_group("obstacle_detector", type="tab")
 
 group_field_color_detector = group_color_detector.add_group("field_color_detector")
 group_white_color_detector = group_color_detector.add_group("white_color_detector")
 group_red_color_detector = group_color_detector.add_group("red_color_detector")
 group_blue_color_detector = group_color_detector.add_group("blue_color_detector")
 
-group_ball_fcnn.add("ball_fcnn_publish_debug_img", bool_t, 0, "toggles publishing of the fcnn heatmap image for debug purposes", None)
-group_ball_fcnn.add("neural_network_model_path", str_t, 0, "neural_network_model_path", fcnn_paths[0], edit_method=fcnn_paths_enum)
-group_ball_fcnn.add("ball_fcnn_threshold", double_t, 0, "ball_fcnn_threshold", min=0.0, max=1.0)
-group_ball_fcnn.add("ball_fcnn_expand_stepsize", int_t, 0, "ball_fcnn_expand_stepsize", min=1, max=20)
-group_ball_fcnn.add("ball_fcnn_pointcloud_stepsize", int_t, 0, "ball_fcnn_pointcloud_stepsize", min=1, max=20)
-group_ball_fcnn.add("ball_fcnn_shuffle_candidate_list", bool_t, 0, "ball_fcnn_shuffle_candidate_list")
-group_ball_fcnn.add("ball_fcnn_min_ball_diameter", int_t, 0, "ball_fcnn_min_ball_diameter", min=1, max=50)
-group_ball_fcnn.add("ball_fcnn_max_ball_diameter", int_t, 0, "ball_fcnn_max_ball_diameter", min=1, max=600)
-group_ball_fcnn.add("ball_fcnn_candidate_refinement_iteration_count", int_t, 0, "ball_fcnn_candidate_refinement_iteration_count", min=1,max=100)
-group_ball_fcnn.add("ball_fcnn_publish_output", bool_t, 0, "publish the output of the ball fcnn as ImageWithCorners", None)
-group_ball_fcnn.add("ball_fcnn_publish_field_boundary_offset", int_t, 0, "the offset added to the field_boundary when cropping the fcnn output for publication in pixels", min=1,max=50)
```

Comment:
Please rename the param `ball_fcnn_publish_output` in order to give a better description of what it does and also rename `ball_fcnn_publish_field_boundary_offset` so that the relation to `ball_fcnn_publish_output` becomes clear.

Reply:
We discussed this naming scheme. Can this be comment be resolved?